docs/usage/config-overview.md

```diff
@@ -0,0 +1,209 @@
+# Renovate Configuration Overview
+
+When Renovate runs on a repository, the final config in use is derived from:
+
+- Global config
+- Inherited config
+- Repository config
+- Resolved presets referenced in config
+
+## Types of Config
+
+### Global config
+
+Global config is the term for config defined by the person or team responsible for running the bot.
+Global config can contain config which is "global only" as well as any configuration options which are valid in Inherited config or Repository config.
+
+If you are an end user of Renovate, such as a user of the Mend Renovate App, then you don't need to care too much about global config other than knowing that there are some settings you cannot change because they are global-only.
+You can skip the rest of this "Global config" section and proceed to "Inherited config".
+
+Global config can be read from file, environment, or CLI parameters.
+You must configure at least one of these in order to provide your bot with the minimum details it needs to run, e.g. credentials.
+
+#### File config
+
+The first place Renovate global config is read from is from file.
```

Comment:
Maybe, it is worth to mention the default values here.

Reply:
How do we ensure those stay up-to-date with the actual values? I'm always a bit wary of adding references to default values/behavior outside of the main "config options docs". Because the config option docs are likely to be updated when the code changes, and other mentions are easily forgotten.

Reply:
I have been reffering it more in the direction of, these are the values first overwriting the default ones and then referencing the configuration options page

Reply:
Mention default values of what?

Reply:
That gobal configs are applied over default values and where to find them.

Reply:
Added